### PR TITLE
Fix issue #9

### PR DESCRIPTION
**Pull request by pa-cholek**
Fix issue #9

### CODE REVIEW

File: src/config.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+     * if max size is not set, original image max width/height will be the constraint;
+     */
+    'max_height' => 1000,
+    'max_width' => 1000,
+    'min_height' => 10,
+    'min_width' => 10,
+    /**
+     * Allow upscale - determines if width or height can be bigger than original image
+     */
+    'allow_upscale' => true,
```
Comment (qunabu) [inline src/config.php:21]: whats the point of `upscale` ? 

it should be disabled always

File: src/Http/Controllers/ImagesController.php [hunk 2]
```diff
@@ -24,16 +26,21 @@ public function __construct(
...
 
-    public function image(Request $request) : RedirectResponse
+    public function image(Request $request): RedirectResponse
     {
         $path = $request->get('path');
         $params = $request->except(['path']);
 
-        $output = $this->imagesService->render($path, $params);
-        return redirect($output['url']);
+        if (RateLimiter::retriesLeft('resize-image:' . $path, $perMinute = 5)) {
```
Comment (qunabu) [inline src/Http/Controllers/ImagesController.php:34]: https://laravel.com/api/8.x/Illuminate/Cache/RateLimiter.html#method_retriesLeft latter says `maxAttempts` nothing to do with `minutes`

File: src/config.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+return [
+    /**
```
Comment (qunabu) [inline src/config.php:4]: have a look here https://github.com/EscolaLMS/settings/issues/7

can you wrap those settings in `public` key ?

File: tests/Api/ImagesTest.php [hunk 4]
```diff
@@ -4,13 +4,14 @@
 
 use Illuminate\Support\Facades\Storage;
```
Comment (qunabu) [inline tests/Api/ImagesTest.php:5]: nice to have 100% CC, it's downgraded now

**Review by qunabu — CHANGES_REQUESTED** [review]

File: src/Http/Controllers/ImagesController.php (hunk 2 above)
Comment (pa-cholek) [reply src/Http/Controllers/ImagesController.php:34]: It depends on decay time, and by default it is a minute (60 seconds)

**Pull request merged**